action.yml

```diff
@@ -31,8 +31,8 @@ runs:
         else 
           status="success"
         fi
-        
-        echo "::set-output name=status::${status}"
+
+        echo "{name}=${status}" >> $GITHUB_OUTPUT"
```

Comment:
@neillfontes-klar Please remove the trailing quote after `$GITHUB_OUTPUT` and make sure the key is `status`, thanks! 

e.g. `echo "status=${status}" >> $GITHUB_OUTPUT`